games-server/minecraft-server/files/minecraft-server.initd-r4

```diff
@@ -0,0 +1,52 @@
+#!/sbin/openrc-run
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+if [ "${SVCNAME}" = "minecraft-server" ]; then
+	instance="main"
+else
+	instance="${SVCNAME#minecraft-server.}"
+fi
+
+dtach_tmpfile="/tmp/${SVCNAME}"
```

Comment:
I am not sure, if that is a good idea, since with that change, we now have predictable tmp file names. I think, this could lead to a security problem. I guess, you need that, in order to make your custom command ```attach``` to work?

Reply:
Indeed, it is for this reason that I have to initialize `dtach_tmpfile` with a predictable file name; I am looking for a better way to create a temporary file while allowing to use the `attach` extra command.

Reply:
@ConiKost I have to admit I am having trouble finding a solution, if there is one…